src/plugins/kibana_usage_collection/server/collectors/kibana/get_saved_object_counts.test.ts

```diff
@@ -16,14 +16,34 @@
  * specific language governing permissions and limitations
  * under the License.
  */
-
+import { elasticsearchServiceMock } from '../../../../../../src/core/server/mocks';
 import { getSavedObjectsCounts } from './get_saved_object_counts';
 
+export function mockGetSavedObjectsCounts(params: any) {
+  const esClient = elasticsearchServiceMock.createClusterClient().asInternalUser;
+  esClient.search
+    // @ts-ignore we only care about the response body
```

Comment:
NIT: This ts-ignore is not needed 🙂

Reply:
@afharo when I remove the `@ts-ignore` I get a typescript error:
```
Argument of type '{ body: any; }' is not assignable to parameter of type 'ApiResponse<unknown, unknown> | TransportRequestPromise<ApiResponse<unknown, unknown>>'.
  Type '{ body: any; }' is missing the following properties from type 'ApiResponse<unknown, unknown>': statusCode, headers, warnings, metats(2345)
```